examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<ol><li>Remove "This data is ..."</li><li>Add description of the levels in the dataset, explain how the column names in the dataset are formed</li><li>It would be perfect if you add picture of the hierarchy </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>